vmdb/app/models/miq_provision_microsoft_workflow.rb

```diff
@@ -0,0 +1,63 @@
+class MiqProvisionMicrosoftWorkflow < MiqProvisionInfraWorkflow
+  def self.default_dialog_file
+    'miq_provision_dialogs'
```

Comment:
This is defined in the parent class and you are not overriding it so it is not needed.